### DISCUSSION TITLE
Pausing play while moving the region

### DISCUSSION THREAD

**Discussion by vranicf** [katspaugh/wavesurfer.js#3382] [Q&A] [answered]
Hi,

Is it possible to pause audio reproduction while moving the region? Would that be useful? What would happen if the cursor sat outside the region after moving? Thanks.

**Comment by katspaugh** [comment, marked as answer]
You can do it in your app by subscribing to the region drag event and pausing in the callback. Whether it's gonna be useful is up to the requirements of your app.

**Comment by vranicf** [comment]
Thanks for the quick response!
